src/mon/OSDMonitor.cc

```diff
@@ -1087,6 +1087,11 @@ void OSDMonitor::encode_pending(MonitorDBStore::TransactionRef t)
 
     // clean inappropriate pg_upmap/pg_upmap_items (if any)
     osdmap.maybe_remove_pg_upmaps(cct, osdmap, tmp, &pending_inc);
+    // throw out any stale upmap items too
+    // note that the above check may have deleted some upmap items already
+    // it shouldn't matter here since __clean_pg_upmaps__ is basically doing
```

Comment:
just curious why you are quoting `clean_pg_upmaps` with `__` in code and in commit message. because you are using an editor which renders it differently?

Reply:
Not really. It's simply because I am a little bit getting used to the markdown styling now.
There are some examples [1] which I might be learned from, though.

----
[1] https://github.com/ceph/ceph/blob/master/src/crush/builder.h